cfgov/v1/documents.py

```diff
@@ -0,0 +1,201 @@
+from django_elasticsearch_dsl import Document, fields
+from django_elasticsearch_dsl.registries import registry
+
+from v1.models.base import (
+    CFGOVAuthoredPages, CFGOVPageCategory, CFGOVTaggedPages
+)
+from v1.models.learn_page import AbstractFilterPage
+
+
+@registry.register_document
+class FilterablePagesDocument(Document):
+
+    tags = fields.ObjectField(properties={
+        'slug': fields.KeywordField()
+    })
+    categories = fields.ObjectField(properties={
+        'name': fields.KeywordField()
+    })
+    authors = fields.ObjectField(properties={
+        'name': fields.TextField(),
+        'slug': fields.KeywordField()
+    })
+    title = fields.TextField(attr='title')
+    is_archived = fields.KeywordField(attr='is_archived')
+    content = fields.TextField()
+    date_published = fields.DateField(attr='date_published')
+    url = fields.KeywordField()
+    start_dt = fields.DateField()
+    end_dt = fields.DateField()
+    statuses = fields.KeywordField()
+    initial_filing_date = fields.DateField()
+
+    def get_queryset(self):
+        return AbstractFilterPage.objects.live().public()
```

Comment:
If you add [`.specific()`](https://docs.wagtail.io/en/stable/reference/pages/queryset_reference.html?#wagtail.core.query.PageQuerySet.specific) here, it should pull back all specific instances, avoiding the need to do `instance.specific` several times down below.